src/components/activities/editPatientActivity/EditPatientActivity.tsx

```diff
@@ -143,6 +143,7 @@ const EditPatientActivity: FunctionComponent<TProps> = ({
                 isLoading={isLoading}
                 shouldResetForm={shouldResetForm}
                 resetFormCallback={resetFormCallback}
+                mode = {'edit'}
```

Comment:
There's still something funny here. It looks like prettier is not formatting the code somehow.
```suggestion
                mode={"edit"}
```